patch_docs.py

```
@@ -0,0 +1,103 @@
+        " *       500:",
+        " *         description: Internal server error",
+        " */"
+    ])
+    
+    return "\n".join(doc) + "\n"
+
+for filepath in files:
+    with open(filepath, 'r', encoding='utf-8') as f:
+        try:
+            lines = f.readlines()
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)